OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile.xe

```diff
@@ -0,0 +1,87 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2018, 2021 Oracle and/or its affiliates.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Database 21c Express Edition
+# 
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# None
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Run: 
+#      $ docker build -t oracle/database:21.4.0-xe -f Dockerfile.xe .
+#
+#
+# Pull base image
+# ---------------
+FROM oraclelinux:7-slim
```

Comment:
Question: should we use OL7 or OL8? I don't mind either way just wanted to bring it up. It should be the same that we use for EE/SE2, of course, first and foremost.